src/components/form/Form.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import theme from "@/themes/theme";
+
+const Textarea = ({ className, children, id, name, onSubmit, onReset }) => (
```

Comment:
The component should be named `Form`, not `Textarea`.

Reply:
yeah 😅, sorry

Reply:
No worries :)